src/olympia/amo/tests/test_redirects.py

```diff
@@ -37,13 +37,6 @@ def test_utf8(self):
         # Sphinx will be off so let's just test that it redirects.
         assert response.redirect_chain[0][1] == 302
 
-    def test_parameters(self):
-        """Bug 554976. Make sure when we redirect, we preserve our query
-        strings."""
-        url = u'/users/login?to=/en-US/firefox/users/edit'
-        r = self.client.get(url, follow=True)
-        self.assert3xx(r, '/en-US/firefox' + url, status_code=302)
-
```

Comment:
I tried to find an appropriate replacement for this, but I don't think there is one (excluding legacy views that will be removed in a few weeks).  The original bug is from 9 years ago and doesn't link to the commit so I can't 100% see what was going on anyway.